grails5 upgrade - geb issue

niravassar opened the pull request.

Commit:
grails5 upgrade

sdelamo commented on complete/build.gradle:
```diff
@@ -5,7 +5,7 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails Version

sdelamo commented on complete/build.gradle:
```diff
@@ -5,7 +5,7 @@ buildscript {
+        classpath "org.grails.plugins:hibernate5:7.1.0"
         classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
         //tag::buildscriptwebdriverbinaries[]
         classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.1" // <1>
```
this should be 2.4

sdelamo commented on complete/build.gradle:
```diff
@@ -40,51 +40,54 @@ configurations {
 //tag::webdriverBinaries[]
 webdriverBinaries {
-    chromedriver "${chromeDriverVersion}"  // <6>
+    chromedriver {
```
this should be the same version as grails_i18n guide

sdelamo commented on complete/build.gradle:
```diff
@@ -40,51 +40,54 @@ configurations {
+        version = "${chromeDriverVersion}"
+        architecture = "X86"
+    }  // <6>
     geckodriver "${geckodriverVersion}" // <7>
```
this should be the same version as grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,7 +1,11 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
-assetPipelineVersion=3.0.11
+grailsVersion=5.0.0
```
5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,7 +1,11 @@
-gormVersion=7.0.2.RELEASE
-assetPipelineVersion=3.0.11
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
7.1.0

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,7 +1,11 @@
 chromeDriverVersion=78.0.3904.105
-geckodriverVersion=0.26.0
\ No newline at end of file
+geckodriverVersion=0.26.0
```
this should be the same as grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,7 +1,11 @@
+assetPipelineVersion=3.3.4
 webdriverBinariesVersion=1.4
 seleniumVersion=3.141.59
 chromeDriverVersion=78.0.3904.105
```
this should be the same as grails_i18n guide

sdelamo commented on initial/build.gradle:
```diff
@@ -5,9 +5,11 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails Version

sdelamo commented on initial/build.gradle:
```diff
@@ -5,9 +5,11 @@ buildscript {
+        classpath "org.grails.plugins:hibernate5:7.1.0"
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
+        //tag::buildscriptwebdriverbinaries[]
+        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.1" // <1>
```
this should be 2.4

sdelamo commented on initial/build.gradle:
```diff
@@ -36,52 +40,57 @@ configurations {
 webdriverBinaries {
-    chromedriver '78.0.3904.105'
-    geckodriver '0.26.0'
+    chromedriver {
```
this should be the same version as grails-i18n guide

sdelamo commented on initial/build.gradle:
```diff
@@ -36,52 +40,57 @@ configurations {
+        version = "${chromeDriverVersion}"
+        architecture = "X86"
+    }  // <6>
+    geckodriver "${geckodriverVersion}" // <7>
```
this should be the same version as grails-i18n guide

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,11 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
5.0.1

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,11 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
7.1.0

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,11 @@
+org.gradle.parallel=true
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
+assetPipelineVersion=3.3.4
+webdriverBinariesVersion=1.4
```
2.4 update the plugin to use github webdriver-binaries plugin

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,11 @@
+assetPipelineVersion=3.3.4
+webdriverBinariesVersion=1.4
+seleniumVersion=3.141.59
+chromeDriverVersion=78.0.3904.105
```
use same version as grails_i18n guide

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,11 @@
+webdriverBinariesVersion=1.4
+seleniumVersion=3.141.59
+chromeDriverVersion=78.0.3904.105
+geckodriverVersion=0.26.0
```
use same version as grails_i18n guide

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Fix README canonical URL: use guide version 4 not grailsVersion 5.0.0 (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
